src/routes/(app)/project/[id]/[promptId]/+page.server.ts

```diff
@@ -55,7 +61,8 @@ export async function load({ locals: { supabase }, params }) {
 		instancesReq.data || [],
 		promptsRes.data[0]
 	);
-	const suggestions = getSuggestions(supabase, promptsRes.data[0], false);
+	const hypertuneRoot = await getRootNode(session.user);
```

Comment:
This doesn't change for every request right? Could we set this as a global or something like supabase? expensive to call every time.

Reply:
jk it's per user and it's a REST endpoint. maybe too complicated to cache somehow